src/components/ChartJsWrapper/index.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
Why is this rule being disabled here?

Reply:
There was some error so I disabled this rule to fix. I forgot to enable it. 
Updated!